### DISCUSSION TITLE
Pre alpha

### DISCUSSION THREAD

**Discussion by gdbarros94** [gdbarros94/gerenciamento-Escolar#47] [General]
Version pre alpha with bugs to fix. check issues to know them. 

<hr /><em>This discussion was created from the release <a href='https://github.com/gdbarros94/gerenciamento-Escolar/releases/tag/pre-alpha'>Pre alpha </a>.</em>

**Comment by joaoportolan93** [comment]
Lista das variaveis de cada janela:

sistema de cadastro

Usuario:
Nome
Cargo
E-mail
Senha

Professor:
Nome do Professor
Área
Carga Horária (horas/semana)
Tipo de Contrato
Disponibilidade
Senha

Recursos Alugáveis:
Quantidade
Identificação
Status

Cadastro de Prédios:
nomePredio
andaresPredio
corPredio

Cadastro de Salas:
nomeSala
capacidadeSala
tipoSala
corSala

Cadastro de Andares:
numeroAndar
predioAndar
corAndar

login:
email
password
next

**Comment by gdbarros94** [comment]
resolver issues #38  e #39

**Comment by gdbarros94** [comment]
{
nomeSala : "valor",
capacidadeSala
tipoSala
corSala
}
